drivers/usb/device/Kconfig

```diff
@@ -43,7 +43,7 @@ config USB_DC_RPI_PICO
 DT_STM32_USBHS := $(dt_nodelabel_path,usbotg_hs)
 DT_STM32_USBHS_SPEED := $(dt_node_str_prop_equals,$(DT_STM32_USBHS),maximum-speed,high-speed)
 
-config USB_DC_STM32
+menuconfig USB_DC_STM32
```

Comment:
I do not think it makes anything better here, there is only one child and then it is inconsistent.